pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -312,7 +312,8 @@ def get_evaluate_epoch_results(self, test_mode):
 
         # clear mem
         self.eval_loop_results = []
-        return results
+
```

Comment:
comment to the `get_evaluate_epoch_results` as it is possible to attach it to func name...
when is this used? tracing this func is a bit hell as it resigned on multiple places (I gave up after two)

Reply:
I don't understand what you mean.